src/components/DeploymentView/DeploymentView.js

```diff
@@ -80,6 +120,41 @@ class DeploymentView extends React.PureComponent<Props> {
         SMART_WALLET_UPGRADE_STATUSES.TRANSFERRING_ASSETS,
       ].includes(smartWalletStatus.status);
 
+    let detailedTransaction;
+    const matchingTransaction = history.find(({ hash }) => hash === 'x');
```

Comment:
How this could be possible? I mean to get a tx with the hash equal 'x'?

Reply:
Committed by mistake, fixed.